packages/webcert/src/feature/certificate/CertificateSidePanel/AboutCertificatePanel.tsx

```diff
@@ -55,6 +56,41 @@ const AboutCertificatePanel: React.FC<Props> = ({ headerHeight }) => {
     setShouldLimitHeight(node ? node.scrollHeight > node.clientHeight : false)
   }, [])
 
+  const hasDynamicLink = (text?: string): boolean => {
+    if (!text) {
+      return false
+    }
+    return text.split('<LINK:').length > 1
+  }
+
+  const formatText = (text?: string) => {
+    if (!text) {
+      return ''
+    }
+    const splitText = text.split('<LINK:')
+    if (splitText.length > 1) {
+      let returnValue = (
+        <>
+          <span dangerouslySetInnerHTML={sanitizeText(splitText[0])}></span>
+        </>
+      )
+
+      for (let i = 0; i < splitText.length - 1; i++) {
+        const dynamicLinkKey = splitText[i + 1].split('>')[0]
+        const textAfterLink = splitText[i + 1].substring(splitText[i + 1].indexOf('>') + 1)
+        returnValue = (
+          <>
+            {returnValue}
+            <WCDynamicLink linkKey={dynamicLinkKey} />
+            <span dangerouslySetInnerHTML={sanitizeText(textAfterLink)}></span>
+          </>
+        )
+      }
+      return <>{returnValue}</>
+    }
+    return text
+  }
+
```

Comment:
This is also used in CertificateListRow, so we should probably move this out to a utility function. That would also make writing tests for this function easier.

Reply:
I've moved it out now, and also updated CertificateListRow accordingly